packages/nimble-components/src/patterns/tab/styles.ts

```diff
@@ -36,12 +36,12 @@ export const styles = css`
         background-color: ${fillHoverColor};
     }
 
-    :host(:focus) {
-        outline: none;
+    :host(:hover[aria-selected='true']) {
```

Comment:
This was a bug I found with the existing tab styling: it was showing the selected hover color (green) for the focused tab rather than the selected tab.